cosmovisor/cmd/cosmovisor/cmd/help.go

```diff
@@ -17,20 +13,13 @@ var HelpArgs = []string{"help", "--help", "-h"}
 // Help is needed if either cosmovisor.EnvName and/or cosmovisor.EnvHome env vars aren't set.
 // Help is requested if the first arg is "help", "--help", or "-h".
 func ShouldGiveHelp(arg string) bool {
-	return isOneOf(arg, HelpArgs) || len(os.Getenv(cosmovisor.EnvName)) == 0 || len(os.Getenv(cosmovisor.EnvHome)) == 0
+	return isOneOf(arg, HelpArgs)
```

Comment:
the reason you have seen the help message when running `cosmovisor run start` instead of an error was because of this env checks. You are right, it was confusing to not see actual error. I removed them